src/promptflow/tests/executor/unittests/contracts/test_multimedia.py

```diff
@@ -20,4 +20,12 @@ def test_constructors(self):
             assert b._mime_type == "image/*"
             assert b._hash == "a94a8fe5"
             assert b.to_base64() == "dGVzdA=="
+            assert b.to_base64(True) == "data:image/*;base64,dGVzdA=="
+            assert b.to_base64(True, True) == {"data:image/*;base64": "dGVzdA=="}
```

Comment:
Can we call these with argument names? Otherwise, we can't tell what each `True` stands for.
```suggestion
            assert b.to_base64(with_type=True, dict_type=True) == {"data:image/*;base64": "dGVzdA=="}
```